src/snippets/product-card.liquid

```diff
@@ -1,13 +1,13 @@
 {% unless grid_item_width %}
   {% assign grid_item_width = 'col-sm-6 col-lg-3' %}
+  {% assign vendor = vendor %}
```

Comment:
@rdnvol I think it's better to rename the variable you pass to display_vendor, then it won't be necessary.

Reply:
@IhorTretiy 
I forgot to remove this variable 
It's no longer necessary. 
I'll remove it.